tutorial/05-redux-immutable-fetch.md

```diff
@@ -75,15 +75,16 @@ This file exposes an *action*, `SAY_HELLO`, and its *action creator*, `sayHello`
 ```js
 // @flow
 
-import * as Immutable from 'immutable'
+import Immutable from 'immutable'
+import type { fromJS } from 'immutable'
 
 import { SAY_HELLO } from '../action/hello'
 
 const initialState = Immutable.fromJS({
   message: 'Initial reducer message',
 })
 
-const helloReducer = (state: Object = initialState, action: { type: string, payload: any }) => {
+const helloReducer = (state: fromJS = initialState, action: { type: string, payload: any }) => {
```

Comment:
Let's rename this type to make it clearer:

`import type { fromJS as Immut } from 'immutable'`

`state: Immut`